common/buildcraft/transport/plug/FacadeBlockStateInfo.java

```diff
@@ -36,7 +37,7 @@ public FacadeBlockStateInfo(IBlockState state, ItemStack requiredStack,
         this.isVisible = !requiredStack.isEmpty();
         IBlockAccess access = new SingleBlockAccess(state);
         for (EnumFacing side : EnumFacing.VALUES) {
-            isSideSolid[side.ordinal()] = state.isSideSolid(access, BlockPos.ORIGIN, side);
+            isSideSolid[side.ordinal()] = state.isSideSolid(access, BlockPos.ORIGIN, side) || state.getBlockFaceShape(access, BlockPos.ORIGIN, side) == BlockFaceShape.SOLID;
```

Comment:
What states does this make a difference for?

Reply:
But none of those have solid sides. (Solid -- can place levers, rails, etc on the top or sides).
It might make more sense to add a ````BlockFaceShape```` property.